tests/src/test/scala/scalaz/IListTest.scala

```diff
@@ -287,6 +287,14 @@ object IListTest extends SpecLite {
     ns.prefixLength(f) must_=== ns.toList.prefixLength(f)
   }
 
+  "prepend" ! forAll { (ns: IList[Int], ms: IList[Int]) =>
+    (ns ++: ms).toList must_== ns.toList ++: ms.toList
+  }
```

Comment:
Seems like the exact same test is already on [line 100](https://github.com/scalaz/scalaz/pull/2063/files#diff-5b63ab3c5a167467caefa53a0c7d5eedR100).

Reply:
Same as above, I wrote it to be sure `prepend` and `++=` should be indistinguishable.